tests/small/LandIce/SHMIP/input_suite_A1.yaml

```diff
@@ -65,6 +65,8 @@ ANONYMOUS:
         Type: Given Field
     LandIce Basal Friction Coefficient: 
       Type: Regularized Coulomb
+      Mu Type: Uniform
```

Comment:
Off topic: this is a somewhat silly thing I did: the Hydrology pb always builds the BasalFrictionCoefficient evaluator, so this sublist needs to be there. However, these tests are not using the evaluator. So maybe I should (at some point) change the Hydrology pb, so that it doesn't build the ev if basal melting is not used (so that I can remove these sublists).